paths.go

```diff
@@ -44,12 +44,11 @@ func newPaths(os, arch string, o Options) (p *Paths, err error) {
 	p.initAstilectronDirectory()
 	p.astilectronApplication = filepath.Join(p.astilectronDirectory, "main.js")
 	p.astilectronDownloadSrc = AstilectronDownloadSrc()
-	p.astilectronDownloadDst = filepath.Join(p.vendorDirectory, fmt.Sprintf("astilectron-v%s.zip", VersionAstilectron))
+	p.astilectronDownloadDst = filepath.Join(p.vendorDirectory, fmt.Sprintf("astilectron-%s-%s-v%s.zip", os, arch, VersionAstilectron))
 	p.astilectronUnzipSrc = filepath.Join(p.astilectronDownloadDst, fmt.Sprintf("astilectron-%s", VersionAstilectron))
 	p.electronDirectory = filepath.Join(p.vendorDirectory, "electron")
```

Comment:
@asticode I think electron is enough since the dir will be created only in one OS. I have using this feature in our team's project. Some body using MacOS and others Windows. It's work great.